component.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Component
+spec:
+  type: scheduler
```

Comment:
From a [previous discussion](https://github.com/kubernetes/kubernetes/pull/12324#discussion_r36377594) on replicated components:
> When you do a "create" it generates a name like scheduler-12345 based on the Type. It returns this name so that the caller can use it as a unique identifier, for updates.

From a [previous comment](https://github.com/kubernetes/kubernetes/pull/12324#issuecomment-129042823) on presence monitoring:
> We did also talk about maybe having the external deployer of the cluster (manual or automated) do the "create" call, to initialize the apiserver with the expected components. That way you have a full list of everything that's supposed to come up, even if they are improperly configured or DOA.